compiler+runtime/src/cpp/jank/runtime/obj/detail/base_persistent_map.cpp

```diff
@@ -1,7 +1,12 @@
+#include <fmt/format.h>
+#include <jank/native_persistent_string/fmt.hpp>
```

Comment:
I see the reasoning for grouping these, but that's not how we roll. Please keep the jank includes together, with fmt above them, separated by a new line.